build.gradle.kts

```diff
@@ -169,6 +169,14 @@ dependencies {
         exclude(group = "org.jetbrains", module = "annotations")
         exclude(group = "org.checkerframework", module = "checker-qual")
     }
+    implementation(libs.adventure.serializerAnsi) {
+        exclude(group = "org.jetbrains", module = "annotations")
+        exclude(group = "org.checkerframework", module = "checker-qual")
+    }
+    implementation(libs.ansi) {
```

Comment:
Do we need to declare `ansi` directly? Shouldn't it be brought in as a transitive dep?